### PR TITLE
Trend pytmv1 0.6.1

### PR DESCRIPTION
**Pull request by shaqnawe**
## Status
In Progress.

## Related Content Pull Request
https://github.com/demisto/content/pull/30157

## Related Issues
Related: link to the issue

## Description
Created new demisto-docker image with custom pytmv1 package.

### CODE REVIEW

**Comment by JudahSchwartz** [comment]
do we need to lock on this specifc version? If yes we should lock in the dependabot config also by telling dependabot to ignore it.
You can find other examples in .github/dependabot.yml

**Comment by JudahSchwartz** [comment]
would reccomend locking on * and using [dynamic versioning](https://github.com/demisto/dockerfiles#dynamic-versioning) to express the version in the tag

**Comment by ilappe** [comment]
@JudahSchwartz 
i recommend to add the module `pytmv1` to the our new image `vendors-sdk`
instead of new image, as this is small module
WDYT?

**Comment by ilappe** [comment]
Hi @shaqnawe 
i will add the module `pytmv1` to the dependencies list in the image `vendors-sdk`
in [docker/vendors-sdk/pyproject.toml](https://github.com/demisto/dockerfiles/blob/master/docker/vendors-sdk/pyproject.toml)
so i close this PR

**Comment by shaqnawe** [comment]
> Hi @shaqnawe i will add the module `pytmv1` to the dependencies list in the image `vendors-sdk` in [docker/vendors-sdk/pyproject.toml](https://github.com/demisto/dockerfiles/blob/master/docker/vendors-sdk/pyproject.toml) so i close this PR

Is there something I need to do on this?

**Comment by kgal-pan** [comment]
Looks like we have some hard-coded `requests` versions used by `pytmv1` and `pycti`. Since `pycti` is locked because of https://github.com/demisto/dockerfiles/pull/19991. I think the fastest way forward is to create a new image specifically for `pytmv1` in this PR.

@shaqnawe  - Can you please add a `verify.py` script to fix https://github.com/demisto/dockerfiles/actions/runs/6502596031/job/17714329333?pr=20932? See https://github.com/demisto/dockerfiles?tab=readme-ov-file#adding-a-verifypy-script for instructions.

**Comment by ilappe** [comment]
> > Hi @shaqnawe i will add the module `pytmv1` to the dependencies list in the image `vendors-sdk` in [docker/vendors-sdk/pyproject.toml](https://github.com/demisto/dockerfiles/blob/master/docker/vendors-sdk/pyproject.toml) so i close this PR
> 
> @ilappe Is there something I need to do on this?

Hi 
as mentioned above - we have issue to add `pytmv1` to an existing image so we will need to create a separate image for that 
please follow the comments above 
- https://github.com/demisto/dockerfiles/pull/20932#issuecomment-1763348490
- https://github.com/demisto/dockerfiles/pull/20932#issuecomment-1763348853
- https://github.com/demisto/dockerfiles/pull/20932#issuecomment-1774047322
thanks!

**Comment by shaqnawe** [comment]
@ilappe Please let me know if this is correct, thank you.

File: docker/pytmv1-061/dynamic_version.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+pip freeze | grep demisto-sdk | grep -o -E '[0-9]{1,2}\.[0-9]{1,2}(\.[0-9]+)?'
```
Comment (kgal-pan) [inline docker/pytmv1-061/dynamic_version.sh:1]:
```suggestion
pip freeze | grep pytmv1 | grep -o -E '[0-9]{1,2}\.[0-9]{1,2}(\.[0-9]+)?'
```

**Comment by kgal-pan** [comment]
@shaqnawe - Can you please rename the folder to `pytmv1` (remove `-061` from the name)?

File: docker/pytmv1-061/dynamic_version.sh (hunk 1 above)
Comment (shaqnawe) [reply docker/pytmv1-061/dynamic_version.sh]: done, thank you!

**Comment by ilappe** [comment]
@shaqnawe - thanks!
can you release the lock of `pytmv1 = "==0.6.1"` in the Pipfile as mentioned in [this comment](https://github.com/demisto/dockerfiles/pull/20932#issuecomment-1763348490)?

**Review by ilappe — APPROVED** [review]
Great!

**Comment by shaqnawe** [comment]
There is a new update for pytmv1. Do I need to update the image and push or will that be automatically done?

**Comment by ilappe** [comment]
> @ilappe There is a new update for pytmv1. Do I need to update the image and push or will that be automatically done?

@shaqnawe - we will update it 
@kgal-pan 
can you do re-lock to take the newer version mentioned?

**Comment by kgal-pan** [comment]
> @kgal-pan can you do re-lock to take the newer version mentioned?

https://github.com/demisto/dockerfiles/pull/20932/commits/47e1f4dc5a67afbd59a812f40146eba4ccea689e

**Comment by ilappe** [comment]
will be merged via https://github.com/demisto/dockerfiles/pull/21608

**Pull request closed without merging**